rasa/core/actions/action.py

```diff
@@ -738,8 +738,11 @@ async def run(
             logger.debug(
                 "Calling action endpoint to run action '{}'.".format(self.name())
             )
-            response: Any = await self.action_endpoint.request(
-                json=json_body, method="post", timeout=DEFAULT_REQUEST_TIMEOUT
+            response = await self.action_endpoint.request(
```

Comment:
This var needs the type hint back `Any` so that the code quality mypy check passes.